From 02c8a7920291c53ab07942930c4234c240aff8f1 Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Thu, 6 May 2021 15:32:18 +0200
Subject: [PATCH] Drop duplicate `fillMode` enum definition

The spec is a delta spec and re-defines that enum because it updates the meaning
of one of its values. Patch will likely need to be kept around for as long as
the spec remains a delta spec.
---
 ed/idl/web-animations-2.idl | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ed/idl/web-animations-2.idl b/ed/idl/web-animations-2.idl
index 628282743..bf4f9aa18 100644
--- a/ed/idl/web-animations-2.idl
+++ b/ed/idl/web-animations-2.idl
@@ -33,8 +33,6 @@ partial dictionary ComputedEffectTiming {
     double startTime;
 };
 
-enum FillMode { "none", "forwards", "backwards", "both", "auto" };
-
 [Exposed=Window]
 interface GroupEffect {
   constructor(sequence<AnimationEffect>? children,
-- 
2.31.0.windows.1

